src/components/20-molecules/datepicker/index.scss

```diff
@@ -84,6 +84,7 @@ axa-datepicker {
     position: relative;
     display: flex;
     width: 197px;
+    max-width: 226px;
```

Comment:
the input-field still has an other size

Reply:
Really? The button is moving slightly which is hard to fix, but the outer boundary is no longer moving, which your screenshots showed. Please demonstrate!

Reply:
Input-Field + Button = 217px; datepicker = 217px
<img width="246" alt="Bildschirmfoto 2019-10-02 um 14 19 05" src="https://user-images.githubusercontent.com/38284563/66043599-dc0b8480-e51f-11e9-98ef-453089727acd.png">
Input-Field + Button = 200px; datepicker = 226px
<img width="255" alt="Bildschirmfoto 2019-10-02 um 14 19 23" src="https://user-images.githubusercontent.com/38284563/66043601-dca41b00-e51f-11e9-89e5-dbb164d29cbb.png">

Demonstration
![Bildschirmvideo aufnehmen 2019-10-02 um 14 13 22](https://user-images.githubusercontent.com/38284563/66043370-41ab4100-e51f-11e9-8c4d-9953b0a38cea.gif)

In my opinion input-file + button should be the same size in both examples.

Reply:
OK, the difference in observation is caused by feature-branch pages *not* installing Source Sans Pro fonts (your case) vs my local Storybook having it installed (as required by designers for this component). Not sure whether component must be made robust against a missing required font.